### PR TITLE
added files for RESTful API basic connection

### PR DESCRIPTION
**Pull request by oscarmrt**
added files for RESTful API basic connection

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+app = Flask(__name__)
+app.register_blueprint(app_views)
+
+@app.teardown_appcontext
+def _storage(self):
+    storage.close()
+
+@app.errorhandler(404)
+def not_found(error):
```
Comment (arturovictoriar) [inline api/v1/app.py:18]: comment missing

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+from models import storage
+from api.v1.views import app_views
+from flask import Flask, Blueprint, jsonify, make_response
+
+app = Flask(__name__)
+app.register_blueprint(app_views)
+
+@app.teardown_appcontext
+def _storage(self):
```
Comment (arturovictoriar) [inline api/v1/app.py:12]: comment missing

File: api/v1/views/index.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+from api.v1.views import app_views
+from flask import Flask, jsonify
+from models import storage
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    return jsonify({'status': 'OK'})
+
+@app_views.route('/stats', strict_slashes=False)
```
Comment (arturovictoriar) [inline api/v1/views/index.py:15]: comment missing

**Review by arturovictoriar — CHANGES_REQUESTED** [review]

**Comment by oscarmrt** [comment]
Changes already done on comments for every file.

**Review by arturovictoriar — APPROVED** [review]

**Pull request merged**